lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -124,23 +155,26 @@ def va_profile_opt_in_out_lambda_handler(event: dict, context) -> dict:
 
         try:
             params = (                             # Stored function parameters:
-                record["VaProfileId"],             #     _va_profile_id
-                record["CommunicationItemId"],     #     _communication_item_id
-                record["CommunicationChannelId"],  #     _communication_channel_name
+                record["vaProfileId"],             #     _va_profile_id
+                record["communicationItemId"],     #     _communication_item_id
+                record["communicationChannelId"],  #     _communication_channel_name
                 record["allowed"],                 #     _allowed
                 record["sourceDate"],              #     _source_datetime
             )
 
             if db_connection is None or db_connection.status != 0:
                 # Attempt to (re-)establish a database connection
-                db_connection = make_connection()
+                db_connection = make_connection(worker_id)
 
             if db_connection is None:
                 raise RuntimeError("No database connection.")
 
             # Execute the stored function.
             with db_connection.cursor() as c:
-                put_record["status"] = "COMPLETED_SUCCESS" if c.execute(OPT_IN_OUT_QUERY, params) else "COMPLETED_NOOP"
+                # https://www.psycopg.org/docs/cursor.html#cursor.execute
+                c.execute(OPT_IN_OUT_QUERY, params)
```

Comment:
We dont want to run the query if communicationitemId is 5, right?  put_record["status"] should be COMPLETED_NOOP if communicationItemId != 5

Reply:
It should be checked prior to entering the `with` block, but that's a different ticket.